InputValidation/ValidationMethods/Validation.FirstCharacter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using TotalImage.Properties;
+using TotalImage.Validation;
+
+/// <summary>
+/// Validates the first character of a DOS 8.3 filename (in the case it has been overwritten with 0xE5) (2020/08/01)
+/// </summary>
+namespace TotalImage.Validation.ValidationMethods
+{
+    public class FirstCharacterValidation : IValidation
+    {
+        public object Validate(object Data) 
+        {
+            // Return if not char
+            Data = ValidationCore(Data);
+
+            if (Data == null)
+            {
+                OnFail();
+            }
+            else
+            {
+                return Data;
+            }
+
+            //blah
+            return null; 
+
+        }
+
+        private object ValidationCore(object Data)
+        {
+            if (!VerifyDataType(Data))
```

Comment:
Not sure in the value of calling out to a separate method for this, just inlining it as the following
```c#
if (!(Data is char D))
{
   return null;
}
```

is probably better, and also lets you avoid the cast on Line 45.

Reply:
I need to stop being obsessed with creating useless methods .